ipywidgets/static/widgets/js/widget_box.js

```diff
@@ -16,8 +16,74 @@ define([
         }, widget.WidgetModel.serializers)
     });
 
+    var ProxyModel = widget.WidgetModel.extend({}, {
+        serializers: _.extend({
+            child: {deserialize: widget.unpack_models},
+        }, widget.WidgetModel.serializers),
+    });
+
+    var ProxyView = widget.WidgetView.extend({
+        initialize: function() {
+            // Public constructor
+            ProxyView.__super__.initialize.apply(this, arguments);
+            this.$box = this.$el;
+            this.child_promise = Promise.resolve();
```

Comment:
Hum, this is something we did when we started using Promises in ipywidgets, i think to prevent order problems when setting the child multiple times.